### ISSUE TITLE
Create SBOM one page overview

### ISSUE THREAD

**Issue by joshbressers** [ossf/sbom-everywhere#4]
Kate Stewart volunteered to create a one page overview and simple guidance for SBOM

**Comment by kestewart** [comment]
Yes, I'm working on it, and will post a link to the draft here for others to review.

**Comment by joshbressers** [comment]
Also this page has some good questions
https://cwiki.apache.org/confluence/display/COMDEV/SBOM

**Comment by joshbressers** [comment]
We probably want to note the difference between the source and binary SBOM content

**Comment by joshbressers** [comment]
The draft document can be found here

https://docs.google.com/presentation/d/1L6UawJXIFmZnqX8gLH4eTNpfx1OVgOLoN2GYV5GsLIM/edit#slide=id.g14917e2ac23_0_0

**Comment by joshbressers** [comment]
I'm going to close this as it's very stale now
